src/encode_lib_log_parser.py

```diff
@@ -0,0 +1,624 @@
+#!/usr/bin/env python
+"""
+ENCODE QC log parser wrapper which converts a log file into a dict
+
+Author: Jin Lee (leepc12@gmail.com)
+"""
+
+from collections import OrderedDict
+
+
+MAP_KEY_DESC_FRAC_MITO_QC = {
+    'non_mito_reads': 'Rm = Number of Non-mitochondrial Reads',
+    'mito_reads': 'Rn = Number of Mitochondrial Reads',
+    'frac_mito_reads': 'Rm/(Rn+Rm) = Frac. of mitochondrial reads'
+}
+
+def parse_frac_mito_qc(txt):
+    result = OrderedDict()
+    with open(txt, 'r') as fp:
+        for line in fp.read().strip('\n').split('\n'):
```

Comment:
I will keep it as is. It should be actually `for line in fp.read().strip('\n').splitlines():` because I wanted to remove a possible new line (empty) at the end. There is actually no difference.